### DISCUSSION TITLE
wire:loading not working with Livewire + Laravel Vite + Bootstrap 5.2

### DISCUSSION THREAD

**Discussion by LexanderStacks** [livewire/livewire#5605] [1. Help] [answered]
Technology Stack: Laravel 9 with Vite, Jetstream, Livewire, Bootstrap 5.2, AlpineJS

Body:

```
<div> 
   <button wire:click="activeUser">Show</button>
   <div wire:loading wire:target="activeUser"> Processing ... </div> 
</div>

```
In the Controller there is an example function, which should just be triggered by a click on "Show".

After Loading the View file, there is instantly shown "Processing ..." without any User Interaction - no click.

I am glad about any help, or alternative !!

I tried the example above, and was expecting that "Processing ..." will only be shown after a click on the "Show" Button.

**Comment by joshhanley** [comment, marked as answer]
@LexanderStacks generally if an element with `wire:loading` on it is showing up on initial page load, that is because Livewire's styles are missing or not being rendered to the page. So make sure you have `<livewire:styles />` or `@livewireStyles` in your layout file.

Hope this helps!

**Reply by LexanderStacks** [reply]
@joshhanley 
I already did it, so that it works. I created a new layout and wrote all new - then all worked well. I checked Your answear. I searched for my older layout which was not working and putted it in a diffChecker, comparing it with my new layout which works. In the older one there was the @livewireScripts under the body part but unintentionally I missed there the @livewireStyles line above. Thanks for the answear!

**Reply by joshhanley** [reply]
@LexanderStacks no worries! 🙂
